File: python/prism_sdk/throughput_research_bundle_integrity_contract_model.py
"""Bundle P32 throughput contract_model signed research-object integrity feature."""
from .research_bundle_integrity_support import BundleCard7,BundleReleaseRequest4,ResearchBundleIntegrityError,manifest,release
FEATURE_ID="AFA-bundle-P32-F07";CONTRACT_VERSION="bundle-throughput_research_bundle_integrity_contract_model/1.0"
def throughput_research_bundle_integrity_contract_model_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="contract_model")
def release_throughput_research_bundle_integrity_contract_model(request:BundleReleaseRequest4)->BundleCard7:return release(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="contract_model")
__all__=["FEATURE_ID","CONTRACT_VERSION","throughput_research_bundle_integrity_contract_model_manifest","release_throughput_research_bundle_integrity_contract_model"]

File: crates/bundle/src/throughput_research_bundle_integrity_contract_model.rs
//! Bundle P32 throughput contract_model signed research-object integrity feature.
use super::research_bundle_integrity_support::{manifest,release,BundleCard7,BundleReleaseRequest4,ResearchBundleIntegrityError};
pub const FEATURE_ID:&str="AFA-bundle-P32-F07";pub const CONTRACT_VERSION:&str="bundle-throughput_research_bundle_integrity_contract_model/1.0";
pub fn throughput_research_bundle_integrity_contract_model_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"throughput","contract_model")}
pub fn release_throughput_research_bundle_integrity_contract_model(request:&BundleReleaseRequest4)->Result<BundleCard7,ResearchBundleIntegrityError>{release(request,FEATURE_ID,CONTRACT_VERSION,"throughput","contract_model")}
